### DISCUSSION TITLE
Can't seem to be able to use a GHCR image to run a workflow

### DISCUSSION THREAD

**Discussion by JJ** [community/community#25784] [Actions]
<p>Here’s the workflow</p>
<pre><code class="lang-yaml">name: "Test in a Raku container"
on: [ push, pull_request ]
jobs:
  test:
    runs-on: ubuntu-latest
    container:
      image: ghcr.io/jj/gha-raku:latest
      credentials:
        username: ${{ github.actor }}
        password: ${{ secrets.GITHUB_TOKEN }}
    steps:
      - name: Checkout
        uses: actions/checkout@v2
      - name: Install modules
        run: |
          raku -e 'say $*REPO.repo-chain'
          HOME=/home/raku &amp;&amp; zef install --verbose --deps-only .

      - name: Test
        run: zef --debug test .
</code></pre>
<p>It’s run <a href="https://github.com/JJ/p6-pod-load/runs/4574472980?check_suite_focus=true" rel="noopener nofollow ugc">here</a><br>
with this error</p>
<pre><code class="lang-auto">/usr/bin/docker --config /home/runner/work/_temp/.docker_3a31fecb-bf57-483d-93b9-47e99b2057b0 login ghcr.io -u JJ --password-stdin
  /usr/bin/docker --config /home/runner/work/_temp/.docker_3a31fecb-bf57-483d-93b9-47e99b2057b0 pull ghcr.io/jj/gha-raku:latest
  Error response from daemon: manifest unknown
  Warning: Docker pull failed with exit code 1, back off 6.182 seconds before retry.
</code></pre>
<p>The image is <a href="https://github.com/jj/docker-raku/pkgs/container/raku-gha" rel="noopener nofollow ugc">public</a> and, besides, this repo is granted permission, so I don’t even get why do I need to log in there…</p>

**Comment by airtower-luna** [comment, marked as answer]
<aside class="quote no-group" data-username="JJ" data-post="1" data-topic="218436">
<div class="title">
<div class="quote-controls"></div>
 JJ:</div>
<blockquote>
<p><code>      image: ghcr.io/jj/gha-raku:latest</code></p>
</blockquote>
</aside>
<p>I notice there’s a <a href="https://github.com/JJ/docker-raku/pkgs/container/raku-gha"><code>raku-gha</code></a> image listed on your user packages site. Maybe you mean that?</p>

**Comment by JJ** [comment]
<p>Oh, wowo… :man_facepalming:</p>
<p>Thanks a lot. That was so dumb of me.</p>
